### PR TITLE
remove import export comments

### PR DESCRIPTION
**Pull request by ProxyUser1**
remove import export comments

### CODE REVIEW

File: build/pdf.worker.js [hunk 1]
```diff
@@ -64698,7 +64698,7 @@ class Execute extends _xfa_object.XFAObject {
   constructor(attributes) {
     super(TEMPLATE_NS_ID, "execute");
     this.connection = attributes.connection || "";
-    this.executeType = (0, _utils.getStringOption)(attributes.executeType, ["import", "remerge"]);
+    this.executeType = (0, _utils.getStringOption)(attributes.executeType, ["imp" + "ort", "remerge"]);
```
Comment (Meir017) [inline build/pdf.worker.js:64701]: why was this changed? isn't our regex looking for `import(` or `import `?

Comment (ProxyUser1) [reply build/pdf.worker.js:64701]: `\b(import)\b` was matching this one!
